cluster/addons/limit-range/limit-range.json

```diff
@@ -0,0 +1,20 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "LimitRange",
+  "metadata": {
+    "name": "limits"
+  },
+  "spec": {
+    "limits": [
+    {
+      "type": "Container",
+      "min": {
+        "cpu": "400m"
```

Comment:
100m seems preferred to 400m from my perspective as well.  I think using 400m may actually prevent running certain doc examples in our test environments because things could not schedule single minion setups.  It's relative shares anyway, so 100m should work fine with few pods.